sentry/src/main/java/io/sentry/CombinedScopeView.java

```diff
@@ -435,7 +434,7 @@ public void bindClient(@NotNull ISentryClient client) {
 
   @Override
   public @NotNull ISentryClient getClient() {
-    // TODO checking for noop here doesn't allow disabling via client, is that ok?
+    // TODO [HSM] checking for noop here doesn't allow disabling via client, is that ok?
```

Comment:
Should a customer want to disable Sentry this way, they can provide their own implementation of `ISentryClient` that is noop but not detected as such here.